odoo/addons/test_new_api/tests/test_domain.py

```diff
@@ -52,15 +52,15 @@ def test_empty_char(self):
         self.assertListEqual(EmptyChar.search([('name', 'ilike', 'name')]).mapped('name'), ['name'])
         self.assertListEqual(EmptyChar.search([('name', 'not ilike', 'name')]).mapped('name'), ['', False])
 
-        self.assertListEqual(EmptyChar.search([('name', '=', '')]).mapped('name'), [''])
+        self.assertListEqual(EmptyChar.search([('name', '=', '')]).mapped('name'), ['', False])
         self.assertListEqual(EmptyChar.search([('name', '!=', '')]).mapped('name'), ['name'])
         self.assertListEqual(EmptyChar.search([('name', 'ilike', '')]).mapped('name'), ['name', '', False])
```

Comment:
Can you add some test with the `'!'` to show the inconstency ?

(Should we add a test step in `TestExpression._search` to test the complementary domain ?)

Reply:
It's already tested with the combinations, but I will try to provide something clearer in a next PR.